src/components/screens/InfoCard.js

```diff
@@ -31,7 +31,11 @@ const contactView = (contact) => (
     {!!contact.phone && (
       <InfoBox>
         <Icon xml={phoneIcon(colors.primary)} style={styles.margin} />
-        <TouchableOpacity onPress={() => openLink(`tel:${contact.phone}`)}>
+        <TouchableOpacity
+          accessibilityLabel="Telefonnummer"
```

Comment:
better the phone number itself with (Taste)
and than a hint saying that the call will open. something like "Wechselt zur Telefon-App"